llvm/include/llvm/Support/AMDGPUMetadata.h

```diff
@@ -49,6 +49,11 @@ constexpr uint32_t VersionMajorV5 = 1;
 /// HSA metadata minor version for code object V5.
 constexpr uint32_t VersionMinorV5 = 2;
 
+/// HSA metadata major version for code object V6.
+constexpr uint32_t VersionMajorV6 = 1;
+/// HSA metadata minor version for code object V6.
+constexpr uint32_t VersionMinorV6 = 3;
```

Comment:
Minor nit: there's formally no HSA 1.3, as far as I recall (things stopped at 1.2). Someone looking at this might erroneously infer it exists as a foundation mandated standard (and go look up the documentation), or assume it's coming soonTM. It might be beneficial to call that out here, since it would be difficult to break the current progression / use of "HSA" metadata.

Reply:
@AlexVlx, this "HSA Metadata" is AMD-specific "HSA Metadata", so it is not part of the HSA standards. Maybe updating the comment to mention it is AMD-specific should be done.

I'd also prefer to not update the metadata version unless we change it.